src/libstd/process.rs

```diff
@@ -91,7 +96,11 @@ impl IntoInner<imp::Process> for Child {
     fn into_inner(self) -> imp::Process { self.handle }
 }
 
-/// A handle to a child process's stdin
+/// A handle to a child process's stdin. This struct is used in the [`stdin`]
```

Comment:
I'm not an english native but I'm pretty sure you don't write "process's" but "process'". But maybe I'm wrong so please correct me if this is the case.